lib/birthday_bot.rb

```diff
@@ -35,4 +35,13 @@ def start!
       puts "Today is a day that no one was born"
     end
   end
+
+  def mention ( name )
+    if @config.mention
+      "<@#{ name }>"
+    else
+      name
+    end
+  end
```

Comment:
The earlier version of the database stored the real names of users. The bot wished users with their real names. So, I thought that could be a valid use-case

But like my commit message said, some people might prefer wishing their team-mates by their real names (probably for more family-togetherness) and/or not choosing to spam the users with mentions on the wishes (but that wil probably be taken care of by other team-mates, anyway).